doc/sphinx-guides/source/developers/sql-upgrade-scripts.rst

```diff
@@ -28,7 +28,9 @@ How to Create a SQL Upgrade Script
 
 We assume you have already read the :doc:`version-control` section and have been keeping your feature branch up to date with the "develop" branch.
 
-Create a new file called something like ``V4.11.0.1__5565-sanitize-directory-labels.sql`` in the ``src/main/resources/db/migration`` directory. Use a version like "4.11.0.1" in the example above where the previously released version was 4.11, ensuring that the version number is unique. Note that this is not the version that you expect the code changes to be included in (4.12 in this example). When the previously released version is a patch version (e.g. 5.10.1), use "5.10.1.1" for the first SQL script version (rather than "5.10.1.0.1"). For the "description" you should the name of your branch, which should include the GitHub issue you are working on, as in the example above. To read more about Flyway file naming conventions, see https://documentation.red-gate.com/fd/migrations-184127470.html
+Create a new SQL file in the ``src/main/resources/db/migration`` directory. Name the file to something similar to ``V6.1.0.1.sql``. In this example ``6.1`` represents the current version of Dataverse, with the last digit representing number of the script for that version. Should a newer version be merged while you work on your pull request (PR), you must update your script to a higher number.
```

Comment:
```suggestion
Create a new SQL file in the ``src/main/resources/db/migration`` directory and put a short, meaningful comment at the top. Make the filename something like ``V6.1.0.1.sql``. In this example ``6.1`` represents the current version of Dataverse, with the last digit representing number of the script for that version. (The zero in this example is a placeholder in case the current version has a third number like ``6.1.1``.) Should a newer version be merged while you work on your pull request (PR), you must update your script to the next available number such as ``V6.1.0.2.sql``.
```